docs/interfaces/eigen_google.rst

```diff
@@ -0,0 +1,6 @@
+.. _eigen_google:
+
+Eigen interface by Google
+=========================
+
+The Eigen interface by Google is documented `here <https://github.com/google/osqp-cpp>`_.
```

Comment:
Note that this is not an officially supported Google product. I'd say "developed at Google" is more accurate than "by Google".

Reply:
Sure. I made the fix at https://github.com/oxfordcontrol/osqp/pull/266